### PR TITLE
Runtime webhook

### PR DESCRIPTION
**Pull request by PaulRitter**
To avoid us needing to logdive to find issues.
Triggeres at roundend and sends all runtimes to MoMMI, which then posts them into a #runtimes channel in the coderdiscord
**this needs @PJB3005 to edit the MoMMI config as well as the channel to be created**
also 0% tested cause im not setting up MoMMI myself for such a small chance
[qol]

:cl:
 * rscadd: MoMMI will now post all runtimes at roundend in the coderdiscord

### CODE REVIEW

File: code/modules/error_handler/error_viewer.dm [hunk 1]
```diff
@@ -186,7 +190,7 @@
 		html += " <a href='?_src_=holder;adminplayerobservefollow=[usr_ref]'>Follow</a>"
 		if (istype(usr_loc))
 			html += "<br><b>usr.loc</b>: <a href='?_src_=vars;Vars=\ref[usr_loc]'>VV</a>"
-			html += " <a href='?_src_=holder;adminplayerobservecoodjump=1;X=[usr_loc.x];Y=[usr_loc.y];Z=[usr_loc.z]'>JMP</a>"
+			html += " <a href='?_src_=holder;adminplayerobservecoodjump=;X=[usr_loc.x];Y=[usr_loc.y];Z=[usr_loc.z]'>JMP</a>"
```
Comment (brndd) [inline code/modules/error_handler/error_viewer.dm]: ❓

Comment (PaulRitter) [reply code/modules/error_handler/error_viewer.dm]: oops

**Comment by PJB3005** [comment]
This is a really bad way to implement this. If we *do* want this kind of automatic reporting then MoMMI should fetch the logs from pomf's server automatically and process them manually. Sending each individual runtime through the wire like this is a *mess*.

**Comment by ShiftyRail** [comment]
You can have litteraly thousands of runtimes in a single round
This is a false good idea

**Comment by PJB3005** [comment]
Well, to be fair, it would be sending partially condensed runtime entries, not *each* individual runtime. Still bad though.

**Comment by Kurfursten** [comment]
This requires PJB's backing since it requires him to do stuff to his personal bots. Since it lacks that, I don't see how this can possibly go through.

**Comment by Kurfursten** [comment]
You can let me know if this is something we're going to re-examine

**Comment by PaulRitter** [comment]
probably not, i got more interesting things to work on

**Pull request closed without merging**